src/test/java/com/wikia/webdriver/elements/mercury/components/Category.java

```diff
@@ -31,19 +31,19 @@ public Category(WebDriver driver) {
     PageFactory.initElements(driver, this);
   }
 
-  public Category toggle() {
+  public Category toggleMenu() {
     wait.forElementClickable(categoryMenu);
     categoryMenu.click();
 
-    PageObjectLogging.logInfo("Category was toggled");
+    PageObjectLogging.logInfo("Category component was toggled");
 
     return this;
   }
 
-  public Category openCategoryPage(int index) {
-    WebElement link = categoryList.get(index);
+  public CategoryPage openCategoryPage(String name) {
+    WebElement link = categoryList.findElement(By.cssSelector("a[title=\"" + name + "\"]"));
```

Comment:
how about using `String.format`  - it's much cleaner than string concatenation and you don't have to worry about `"`